Tests/SnapshotTests/Helpers/MockAccountDetailsUseCase.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  MockAccountDetailsUseCase.swift
+//  SnapshotTests
+//
+//  Created by Backbase R&D B.V. on 14/03/2024.
+//
+
+import Foundation
+import AccountsJourney
+import ArrangementsClient2Gen2
+
+
+class MockAccountDetailsUseCase: AccountDetailsUseCase {
+    private var shouldReturnError = false
```

Comment:
= false is not needed, since it's already set from the constructor (init method)